lib/driverProviders/local.ts

```diff
@@ -123,7 +122,12 @@ export class Local extends DriverProvider {
 
     this.server_ = new remote.SeleniumServer(this.config_.seleniumServerJar, serverConf);
 
-    let deferred = q.defer();
+    let deferredResolve: (x?: any) => void;
```

Comment:
Similar to below, we should just return a promise that wraps this code instead of using a deferred